src/main/resources/db/changelog/003-create-table-financial_transactions.yaml

```diff
@@ -18,14 +18,12 @@ databaseChangeLog:
                   type: bigint
               - column:
```

Comment:
Please add: 
constraints:
     nullable: false
without this financial_transaction can be saved with null wallet_id. For our business requirements financial_transaction must know who is his owner

Reply:
Done